kratos/python_scripts/multiple_points_output_process.py

```diff
@@ -83,6 +83,13 @@ def ExecuteBeforeOutputStep(self):
         for proc in self.point_output_processes:
             proc.ExecuteBeforeOutputStep()
 
+    def IsOutputStep(self):
+        return self.point_output_processes[0].IsOutputStep()
```

Comment:
```suggestion
        # We check only the first process as the interval is the same in all of them
        return self.point_output_processes[0].IsOutputStep()
```
Just an annotation for the future.